content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a [month of busy development](https://github.com/OpenRA/OpenRA/pulse/monthly). Time to get back to testing.
```

Comment:
Can we get a static url for this? In a months time the contents of that link will change.